integration/it-integration/build.gradle

```diff
@@ -1,4 +1,5 @@
 dependencies {
+    implementation 'org.projectlombok:lombok:1.18.22'
```

Comment:
From build.gradle in root:

```java
dependencyManagement {
    imports { mavenBom "org.springframework.boot:spring-boot-dependencies:${springbootMavenBomVersion}" }
}
```

This manages our dependencies, so we dont need to specify a version for lombok.